ports/glutin/window.rs

```diff
@@ -384,17 +384,17 @@ impl Window {
                     }
                 };
 
-                if let Ok(key) = Window::glutin_key_to_script_key(virtual_key_code) {
-                    let state = match element_state {
-                        ElementState::Pressed => KeyState::Pressed,
-                        ElementState::Released => KeyState::Released,
-                    };
-                    let modifiers = Window::glutin_mods_to_script_mods(self.key_modifiers.get());
-                    self.event_queue.borrow_mut().push(WindowEvent::KeyEvent(ch, key, state, modifiers));
-                }
-            }
-            Event::KeyboardInput(_, _, None) => {
-                debug!("Keyboard input without virtual key.");
+                let key = match virtual_key_code {
+                    Some(v) => Window::glutin_key_to_script_key(v).unwrap(),
```

Comment:
Because I will remove the ```Result``` from [here](https://github.com/servo/servo/pull/15800/files#diff-f3afdd0d8092731ed5c423ce2e8dfd70R620) but before doing so I prefer to see if you are ok with the changes 😄